db/seeds/activities/future_learn.rb

```diff
@@ -1,4 +1,5 @@
-# This file contains seeds for Activities faciliated by Future Learn
+# This file contains seeds for Activities facilitated by Future Learn, that have since been retired, but we keep in our database
```

Comment:
I'm being picky but `Future Learn` shouldn't have a space ;) `FutureLearn`. May be worth a search in the project to check there are no other remaining instances (there aren't within this PR diff at least)

Reply:
done in https://github.com/NCCE/teachcomputing.org/commit/40e2baa1a49c4ec1be36005dc2a585296b8ea5bf